e2e-network/expect-invoke-tls.sh

```diff
@@ -1,49 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
I'm not a huge fan of this. Using expect-invokes on Fablo Rest is great and simplifies a lot of things, but imo we should still have some raw cli calls with and without tls in tests.
Otherwise we might break something. Invoking/Quering Fabric using cli is different beast and ei. requires proper cert setup in env variables and mounted files